app/src/main/java/ch/epfl/sweng/rps/db/FirebaseReferences.kt

```diff
@@ -12,6 +13,6 @@ open class FirebaseReferences(val env: Env = Env.PROD) {
     fun usersFriendRequestOfUid(uid: String): CollectionReference =
         usersCollection.document(uid).collection("friend_requests")
 
-    val usersCollection = root.collection("users")
+    val usersCollection = FirebaseFirestore.getInstance().collection("users")
```

Comment:
You should not hardcode it like that when we have a root collection. Hardcoding it like that makes it impossible to test with an ENV different than production.